Project/Fall2020_CSC403_Project/FrmInventory.cs

```diff
@@ -0,0 +1,167 @@
+﻿using Fall2020_CSC403_Project.code;
+using MyGameLibrary;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Windows.Forms;
+
+namespace Fall2020_CSC403_Project
+{
+    public partial class FrmInventory : Form
+    {
+        private Inventory inventory;
+        private FrmInventory instance;
+        private Button[] inventorySlots;
+        private Player player;
+        private Thread invUpdater;
+        private bool active;
+
+
+        public FrmInventory(Inventory inventory, Player player)
+        {
+            this.inventory = inventory;
+            this.player = player;
+        }
+
+        // Creates an instance of this class with a visible 
+        // interface for the inventory
+        private void Instance()
+        {
+            InitializeComponent();
+            inventorySlots = new Button[inventory.GetMaxInventorySize()];
+            InitInvSlots();
+            this.FormClosing += new FormClosingEventHandler(FrmIventory_FormClosing);
+            invUpdater = new Thread(ContinuousUpdater);
```

Comment:
There is a bug which I think may be caused by this  where if you open the inventory then close the main game window without closing inventory first, you can no longer rebuild the project as the files will be locked by the still running process in the background which has to be manually killed.